scripts/thanks.js

```diff
@@ -0,0 +1,33 @@
+if (!remoteUrl) {
+  var remoteUrl = "https://api.carpoolvote.com/live";
+}
+
+const handleBackendResponse = () => {
```

Comment:
I love your use of ES6 but we're not transpiling so this will break in some of our supported browsers. Can we please stick to ES5? So please change to 
`var handleBackendResponse = function() {`